backend/app/store/service/service.go

```diff
@@ -493,7 +493,10 @@ func (s *DataStore) HasReplies(comment store.Comment) bool {
 	for _, c := range comments {
 		if c.ParentID != "" && !c.Deleted {
 			if c.ParentID == comment.ID {
-				s.repliesCache.Set(comment.ID, true, cache.DefaultExpiration)
+				// When this code is reached, key "comment.ID" is not in cache.
+				// Calling cache.Get on it will put it in cache with 5 minutes TTL.
+				// We call it with nil value as we care about keys and not values.
+				_, _ = s.repliesCache.Get(comment.ID, func() (lcw.Value, error) { return nil, nil })
```

Comment:
any particular reason why not to set `true` here? I understand we don't care about the value, but nil as a value feels odd. Btw, usually to indicate such "we don't care" `struct{}{}` used

Reply:
Switched to `struct{}{}` as it's guaranteed to take 0 bytes. Both `nil` and `struct{}{}` are less confusing than `true` in my opinion.